pkg/cmd/codespace/delete.go

```diff
@@ -220,6 +220,7 @@ func confirmDeletion(p prompter, apiCodespace *api.Codespace, isInteractive bool
 type surveyPrompter struct{}
 
 func (p *surveyPrompter) Confirm(message string) (bool, error) {
```

Comment:
Could be a followup, but seems like `Confirm` could go to `common.go` next to `Ask` - a little weird here that we're in a `surveryPrompter` method and then building a `Prompter`

Reply:
I missed that, thanks for the call out. Definitely seems like something that should be refactored, so I'll make a note of that to get around to in the future.